__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+
+const db = require('../db/connection.js')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index.js')
+
+const app = require('../app')
+
+beforeEach(() => seed(testData))
+
+afterAll(() => db.end())
+
+describe('GET: /api/topics', () => {
+    test('should return an array of topics with slug and description properties and 200 status', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            body.topics.forEach((topic) => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+            expect(body.topics).toHaveLength(3)
+        })
+    });
+    test('should return a 404 Not Found if given an invalid route', () => {
+        return request(app)
+        .get('/api/topic')
+        .expect(404)
+    });
```

Comment:
Nice :) 
You may want to make this error test separate because it will apply to all endpoints :)